src/components/generic/Modal/Modal.jsx

```diff
@@ -17,47 +14,34 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../../constants';
  * @returns {object}  JSX of the element
  */
 export default function Modal({
-  applicationId,
   children,
   className,
   onClose,
   title,
-  ...props
 }) {
-  const mainClasses = {
-    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-modal`]: true,
-    [className]: !!className,
-  };
-  const testId = `${WEBEX_COMPONENTS_CLASS_PREFIX}-${Math.random().toString().slice(2)}`;
+  const cssClasses = webexComponentClasses('modal-content', className);
 
   return (
-    <MomentumModal
-      applicationId={applicationId}
-      className={classNames(mainClasses)}
-      htmlId={testId}
-      onHide={onClose}
-      show
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...props}
-    >
-      <ModalHeader headerLabel={title} showCloseButton={!!onClose} />
-      <ModalBody>
-        {children}
-      </ModalBody>
-    </MomentumModal>
+    <div className="modal">
```

Comment:
let's call this modal-container and leave the name "modal" for the actual content (what is now named modal-content

it must also be prefixed, otherwise we're likely to get name clashes with other "modal" class defined by devs.

```jsx
<div className={webexComponentClasses('modal-container')}>

Reply:
if you think it's better to have '.modal' for container and '.modal-content' for content, I'm ok with it

Reply:
but it's important to have the top level class of a component prefixed. The rest can be not prefixed.